man/install_cmdstan.Rd

```diff
@@ -4,26 +4,49 @@
 \alias{install_cmdstan}
 \title{Install the latest release of CmdStan}
 \usage{
-install_cmdstan(dir = NULL, cores = 2, quiet = FALSE)
+install_cmdstan(dir = NULL, cores = 2, quiet = FALSE,
+  overwrite = FALSE, repo_clone = FALSE,
+  repo_url = "https://github.com/stan-dev/cmdstan.git",
+  repo_branch = "develop")
 }
 \arguments{
 \item{dir}{Path to the directory in which to install CmdStan. The default is
 to install it in a folder \code{".cmdstanr"} in the user's home directory
 (\code{Sys.getenv("HOME")}).}
 
 \item{cores}{The number of CPU cores to use to parallelize building CmdStan
-and speed up installation.}
+and speed up installation. The default is \code{cores=2}, although we recommend
+using more cores if available.}
 
 \item{quiet}{Defaults to \code{FALSE}, but if \code{TRUE} will suppress the verbose
 output during the installation process.}
+
+\item{overwrite}{Defaults to \code{FALSE}, but if \code{TRUE} will download and
+reinstall CmdStan even if an existing installation is found.}
+
+\item{repo_clone}{Defaults to \code{FALSE}, but if \code{TRUE} will install a
```

Comment:
If \code{FALSE} installs latest CmdStan release from tarball. If \code{TRUE} installs a git clone of CmdStan and checks out the specified branch.